### DISCUSSION TITLE
Transform SCWindow coordinate | Boy's Life

### DISCUSSION THREAD

**Discussion by giscus** [travisbikkle/travisbikkle.github.io#4] [General]
# Transform SCWindow coordinate | Boy's Life

Tips for transforming SCWindow coordinate

https://travisbikkle.pages.dev/en/2024/10/how-did-i-develop-mactoys-pin-window/

**Comment by travisbikkle** [comment]
It's on the App Store now, visit [MagicToy on AppStore](https://apps.apple.com/us/app/magictoy/id6737158347) to try it for free, or visit https://mactoys.eusoftbank.com to learn more 😄
